integreat_cms/google_translate_api/apps.py

```diff
@@ -0,0 +1,118 @@
+"""
+Configuration of Google Translate API app
+"""
+
+from __future__ import annotations
+
+import logging
+import os
+import sys
+from typing import TYPE_CHECKING
+
+from django.apps import AppConfig
+from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+from google.auth.exceptions import DefaultCredentialsError
+from google.cloud import translate_v2, translate_v3  # type: ignore[attr-defined]
+from google.oauth2 import service_account
+
+if TYPE_CHECKING:
+    from typing import Final
+
+    from django.utils.functional import Promise
+
+logger = logging.getLogger(__name__)
+
+
+class GoogleTranslateApiClientConfig(AppConfig):
+    """
+    Google Translate API config inheriting the django AppConfig
+    """
+
+    #: Full Python path to the application
+    name: Final[str] = "integreat_cms.google_translate_api"
+    #: Human-readable name for the application
+    verbose_name: Final[Promise] = _("Google Translate API")
+    #: The supported source languages
+    supported_source_languages: list[str] = []
+    #: The supported target languages
+    supported_target_languages: list[str] = []
+
+    # pylint: disable=too-many-nested-blocks
```

Comment:
Not insisting here, but maybe it actually make sense to move some blocks into separate functions, like ready_v2(), ready_v3() ?

Reply:
Nice idea 😸 applied!